src/CodeExplorerBundle/Resources/config/services.yml

```diff
@@ -1,9 +1,9 @@
 services:
+    _defaults: { public: false }
+
     code_explorer.twig.source_code_extension:
-        public:    false
-        class:     CodeExplorerBundle\Twig\SourceCodeExtension
-        tags:
-            - { name: twig.extension }
+        class: CodeExplorerBundle\Twig\SourceCodeExtension
+        tags: [twig.extension]
```

Comment:
At first I thought as Maxime: let's show the new Symfony in AppBundle and the old Symfony in CodeExplorerBundle. But now I'm starting to think as Christophe. If this is the new and recommended Symfony ... why show the old Symfony too?

Besides, before November we'll delete the CodeExplorerBundle (and Appbundle too!) because we'll use Symfony 4 and there are no bundles in that version!